packages/storybook/stories/ovhcloud-design-system/get-started.stories.mdx

```diff
@@ -38,119 +38,85 @@ You should add @ovhcloud/ods-common-core when you need to use Flag component. Fo
 You should also add @ovhcloud/ods-common-theming when you would like to use a component with an intent.
 
 ### Import components
-Add a dependency to either:
-* `@ovhcloud/ods-components` to import all components
-* `@ovhcloud/ods-components-<component>` to import a specific component
 
-Components can be imported using one of:
-```typescript
-import { Osds<Component> } from '@ovhcloud/ods-component-<component>'
-import { Osds<Component> } from '@ovhcloud/ods-component-<component>/react'
-import { Osds<Component> } from '@ovhcloud/ods-component-<component>/vue'
-```
+#### As native webcomponents
 
-You can import enums directly from the component:
-```typescript
-import { ODS_%COMPONENT%_SIZE, ODS_%COMPONENT%_SIZES } from '@ovhcloud/ods-component-<component>'
-```
+There are two different ways of using ODS webcomponent, using lazy-loading or bundle.
+
+To use lazy-loading, just include in your project the loader script, it will take care of loading each component resources on-demand.
 
-Same goes for interfaces:
 ```typescript
-import type { Ods<Component>Attribute } from '@ovhcloud/ods-component-<component>'
+import { defineCustomElements } from '@ovhcloud/ods-components/dist/loader';
+defineCustomElements();
 ```
 
-`@ovhcloud/ods-component-<component>/react`:
-
-Components are compatible with:
-- React v16.8+
-- TypeScript 3.7+
-
-More information on React generated components with the component builder [stenciljs](https://stenciljs.com/docs/react)
-
-`@ovhcloud/ods-component-<component>/vue`:
+If you prefer to use each components directly, you can use import them separately.
+This is recommended when working on projects bundled (using webpack, rollup, ...) as it is more optimized for tree-shaking.
 
-Components are compatible with:
-- Vue 3
-- TypeScript 4
+```typescript
+import { defineCustomElement as defineButton } from '@ovhcloud/ods-components/dist/components/osds-button';
+defineButton();
+```
 
-More information on Vue generated components with the component builder [stenciljs](https://stenciljs.com/docs/vue)
+#### As React components
 
-#### Loader
+To use a React version of an ODS component, you need to import it from the `react` sub directory.
 
-For frameworks like vue, you can import components via a loader.
-It is also the recommended way if you want IE11 and Edge support with these stencil build made components.
+```typescript
+import { OsdsButtton, OsdsText } from '@ovhcloud/ods-components/react';
```

Comment:
typo: OsdsButtton to OsdsButton
(this file has the same typo throughout, not just here)